README.md

```diff
@@ -74,10 +74,17 @@ After that you can use your jobs as usual.
 
 Specify desired configuration inside of `sidekiq.yml` file:
 
+By default jobs will not be monitored, in order to enable monitoring of your job, you will need to add `:
+attentive_reliable => true` sidekiq option it e.g.
+```ruby
+sidekiq_options :queue => :accessibility, :retry => 3, :attentive_reliable => true
+```
+
+
 ```YML
 attentive:
   # Time in seconds between checks for disappeared jobs
-  :execution_interval: 300  # default: 600
+  :execution_interval: 60  # default: 600
```

Comment:
Might be better to leave this at default and modify it as needed when we import this library into other projects

Reply:
This is only an example in the readme.